packages/fxa-auth-server/lib/devices.js

```diff
@@ -191,6 +191,10 @@ module.exports = (log, db, push) => {
       }
     }
 
+    // No need to await and block the notifications below.  If the records
+    // aren't deleted with this call, they will be once they expire.
+    pushbox.deleteDevice(uid, deviceId);
```

Comment:
In the event a fire and forget is really necessary, having a `.catch((err) => ...)` is good practice. Without this an unhandled rejection can occur which typically crashes the node process.

Reply:
> In the event a fire and forget is really necessary, having a `.catch((err) => ...)` is good practice. Without this an unhandled rejection can occur which typically crashes the node process.

Thanks for the review.  Locally this didn't crash the process, but, Sentry also didn't catch the error like I hope it would.  I'll update it to add a catch and report the error to Sentry in the error handler.